labs/02-Fundementals-DOM-Exercises/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
```

Comment:
Generally really cool always visualing your JS code in HTML, not just through `console`. :)